src/Illuminate/Queue/WorkerOptions.php

```diff
@@ -54,10 +61,11 @@ class WorkerOptions
      * @param  int  $timeout
      * @param  int  $sleep
      * @param  int  $maxTries
+     * @param  int  $maxJobs
      * @param  bool  $force
      * @return void
      */
-    public function __construct($delay = 0, $memory = 128, $timeout = 60, $sleep = 3, $maxTries = 0, $force = false)
+    public function __construct($delay = 0, $memory = 128, $timeout = 60, $sleep = 3, $maxTries = 0, $maxJobs = 0, $force = false)
```

Comment:
You are injecting `$masJobs = 0` - but your not storing the result anywhere?

Reply:
Fixed thanks. Wrote this on the train this morning.